### PR TITLE
video-prop-guidelines

### PR DESCRIPTION
**Pull request by bakatzir**
relates to: https://github.com/demisto/content-docs/pull/1097, https://jira-hq.paloaltonetworks.local/browse/XDR-52801

### CODE REVIEW

File: docs/documentation/pack-docs.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ title: Pack Documentation
 
 Content Packs displayed in the Cortex XSOAR Marketplace contain 2 main documentation sections: 
 * **Description**: displayed in the Content Pack card when browsing the Marketplace and in the top of the *Details* tab.
-* **README**: displayed in the main display area of the *Details* tab.
+* **Video**: displayed in the main display area and in the middle of the *Details* tab.
+* **README**: displayed in the main display area and in the bottom of the *Details* tab.
```
Comment (MosheEichler) [inline docs/documentation/pack-docs.md:9]: Video or Videos?

File: docs/documentation/pack-docs.md [hunk 2]
```diff
@@ -49,6 +50,10 @@ Here are some templates that can help you get started:
 * "Eliminate ________ by improving your__________. Rich with layouts and playbooks, this Content Pack is right for ____________"
 * "Get smarter. This pack utilizes _________ and _______for when _________ is heavily needed"
 
+## Pack Video
```
Comment (MosheEichler) [inline docs/documentation/pack-docs.md:53]: Same here...

File: docs/documentation/pack-docs.md [hunk 3]
```diff
@@ -5,7 +5,8 @@ title: Pack Documentation
 
 Content Packs displayed in the Cortex XSOAR Marketplace contain 2 main documentation sections: 
 * **Description**: displayed in the Content Pack card when browsing the Marketplace and in the top of the *Details* tab.
-* **README**: displayed in the main display area of the *Details* tab.
+* **Video**: displayed in the main display area and in the middle of the *Details* tab.
```
Comment (bakatzir) [inline docs/documentation/pack-docs.md:8]:
```suggestion
* **Videos**: displayed in the main display area and in the middle of the *Details* tab.
```

File: docs/documentation/pack-docs.md (hunk 2 above)
Comment (bakatzir) [inline docs/documentation/pack-docs.md:53]:
```suggestion
## Pack Videos
```

File: docs/documentation/pack-docs.md [hunk 4]
```diff
@@ -49,6 +50,10 @@ Here are some templates that can help you get started:
 * "Eliminate ________ by improving your__________. Rich with layouts and playbooks, this Content Pack is right for ____________"
 * "Get smarter. This pack utilizes _________ and _______for when _________ is heavily needed"
 
+## Pack Videos
+For larger packs that provide at least one end-to-end use case you are encouraged to create a short video for the pack that will be showed in the *Details* tab of the pack in the Marketplace. The video file should hosted in youtube. The video should contain a more detailed overview of the Pack compared to the *Description* section, but stay tuned to the value proposition of the pack.
```
Comment (bakatzir) [inline docs/documentation/pack-docs.md:54]:
```suggestion
For larger packs that provide at least one end-to-end use case you are encouraged to create a short video or a few videos for the pack that will be showed in the *Details* tab of the pack in the Marketplace. The videos files should be hosted in youtube. The videos should contain a more detailed overview of the Pack compared to the *Description* section, but stay tuned to the value proposition of the pack.
```

**Review by MosheEichler — APPROVED** [review]

**Pull request merged**